.swiftlint.yml

```diff
@@ -1,6 +1,7 @@
 disabled_rules:
   - variable_name
   - large_tuple
+  - for_where
```

Comment:
disabling this rule cause `for ... where` looks are strange and we tend not to use `for` loops anyway